guides/source/caching_with_rails.md

```diff
@@ -199,17 +187,92 @@ render(partial: "hotels/hotel", collection: @hotels, formats: :html, cached: tru
 
 Will load a file named `hotels/hotel.html.erb` in any file MIME type, for example you could include this partial in a JavaScript file.
 
-### Managing Dependencies
+### Low-Level Caching using `Rails.cache.fetch`
+
+Sometimes you need to cache a particular value or query result instead of caching view fragments. Rails' caching mechanism works great for storing any serializable information.
+
+The most efficient way to implement low-level caching is using the `Rails.cache.fetch` method. This method handles both _reading from_ and _writing to_ the cache. When called with a single argument, it fetches and returns the cached value for the given key. If a block is passed, the block is executed only on a cache miss. The block's return value is written to the cache under the given cache key and returned. In case of cache hit, the cached value is returned directly without executing the block.
```

Comment:
TIL! I didn't know about `Rails.cache.fetch_multi`, I'll make a note of it so that we can add some details about it in this guide in the future! It sounds useful!